.github/workflows/rasa-sdk-dev-docker-image.yml

```diff
@@ -0,0 +1,59 @@
+name: Create Dev Docker Images
+
+on:
+  schedule:
+    # Run cron job at 8AM Monday to Sunday.
+    - cron: '0 8 * * *'
+  workflow_dispatch:
+
+env:
+  AWS_REGION: us-east-1
+  REPOSITORY: 329710836760.dkr.ecr.us-east-1.amazonaws.com/rasa-sdk
+  DEFAULT_PYTHON_VERSION: "3.10"
+
+permissions:
+  checks: write
+  id-token: write
+  pull-requests: write
+  contents: read
+  issues: read
+
+jobs:
+  rasa-sdk-dev-docker-image:
+    name: rasa-sdk dev docker image
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Read Poetry Version 🔢
+        run: |
+          echo "POETRY_VERSION=$(scripts/poetry-version.sh)" >> $GITHUB_ENV
+        shell: bash
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@8c3f20df09ac63af7b3ae3d7c91f105f857d8497 # v3.0.1
+        with:
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_SESSION_TOKEN }}
```

Comment:
@rasa-aadlv can you please add this secret to the repo?

Reply:
Secret added